components/automate-deployment/pkg/server/api_token.go

```diff
@@ -71,6 +75,21 @@ func generateAdminToken(ctx context.Context,
 		Subjects: []string{fmt.Sprintf("token:%s", response.Id)},
 		Resource: "*",
 	})
+	if err != nil {
```

Comment:
should this just make the above token a member of the admin policy? does that exist yet? 🤔